tests/conftest.py

```diff
@@ -778,11 +792,24 @@ def setup_for_snippet(self, snippet, autostd=True, install_v2_modules: List[Loca
         :param install_v2_modules: Indicates which V2 modules should be installed in the compiler venv
         """
         self.setup_for_snippet_external(snippet)
+        loader.unload_inmanta_plugins()
+        loader.PluginModuleFinder.reset()
```

Comment:
is=n't this done by the project.set call just below?

Reply:
The `reset` isn't but the `unload_inmanta_plugins` is indeed. I'll get rid of that one.

Reply:
(done)